build/commands/lib/util.js

```diff
@@ -722,6 +722,38 @@ const util = {
     return applyPatches()
   },
 
+  updateChromeVersion: () => {
+    const braveVersionParts = config.braveVersion.split('.')
+    assert(braveVersionParts.length == 3)
+
+    const versionFilePath = path.join(config.srcDir, 'chrome', 'VERSION')
+    const versionLines = fs.readFileSync(versionFilePath).toString().split('\n')
```

Comment:
We don't need to use `/\r?\n/`? Chromium git rules makes sure all files always have `\n` line ending?